javascript-modal/main.js

```diff
@@ -0,0 +1,12 @@
+var $openModal = document.querySelector('.modal');
+var $modal = document.querySelector('.absolute');
+var $noButton = document.querySelector('.button-no');
+$openModal.addEventListener('click', toggleModal);
+$noButton.addEventListener('click', toggleModal);
+function toggleModal(event) {
+  if ($modal.className === 'absolute hidden') {
```

Comment:
Instead of making your functionality reliant on the element having a specific class, I would create a variable to keep track of if the modal is open or not, its value should be a boolean.